frontend/ui/src/app/deployments/deployment-wizard/deployment-wizard.component.html

```diff
@@ -0,0 +1,316 @@
+  <!-- Modal header -->
+  <div class="flex items-center justify-between p-4 md:p-5 border-b border-gray-200 rounded-t dark:border-gray-600">
+    <h2 class="text-xl font-semibold text-gray-900 dark:text-white">Create New Deployment</h2>
+    <button
+      type="button"
+      (click)="close()"
+      class="text-gray-400 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm w-8 h-8 ms-auto inline-flex justify-center items-center dark:hover:bg-gray-600 dark:hover:text-white">
+      <fa-icon [icon]="faXmark"></fa-icon>
+      <span class="sr-only">Close modal</span>
+    </button>
+  </div>
+
+  <!-- Modal body -->
+  <div class="p-4 space-y-4 w-full">
+    <section class="antialiased">
+      <div class="mx-auto">
+        <app-deployment-wizard-stepper
+          #stepper
+          (attemptContinue)="attemptContinue()"
+          [showCustomerStep]="showCustomerStep()"
+          class="mx-auto max-w-5xl space-y-6">
+          <!-- Step 1: Customer Selection (Conditional - Only for Vendors) -->
+          @if (showCustomerStep()) {
+            <cdk-step [stepControl]="customerForm">
+              <div class="space-y-6 mt-4">
+                <div class="space-y-4 w-full">
+                  <h3 class="font-normal text-gray-900 dark:text-white">Select Customer</h3>
+                  <p class="text-sm text-gray-500 dark:text-gray-400">
+                    Create the deployment in your vendor organization or choose a customer.
+                  </p>
+
+                  <div class="grid grid-cols-4 gap-3">
+                    <!-- Vendor Only Option -->
+                    @if (vendorOrganization$ | async; as vendorOrg) {
+                      <button
+                        type="button"
+                        (click)="selectCustomer(null)"
+                        class="relative flex flex-col items-center p-4 rounded-lg border-2 transition-all hover:shadow-md"
+                        [class.border-primary-600]="!customerForm.value.customerOrganizationId"
+                        [class.bg-primary-50]="!customerForm.value.customerOrganizationId"
+                        [class.dark:bg-primary-900]="!customerForm.value.customerOrganizationId"
+                        [class.border-gray-200]="customerForm.value.customerOrganizationId"
+                        [class.bg-white]="customerForm.value.customerOrganizationId"
+                        [class.dark:border-gray-700]="customerForm.value.customerOrganizationId"
+                        [class.dark:bg-gray-800]="customerForm.value.customerOrganizationId">
+                        <!-- Vendor Logo -->
+                        @if (vendorBranding$ | async; as branding) {
+                          <img
+                            [src]="getVendorLogoUrl(branding)"
+                            [alt]="vendorOrg.name"
+                            class="w-12 h-12 mb-2 rounded-lg object-cover" />
```

Comment:
logo is cut off 

<img width="800" height="501" alt="Image" src="https://github.com/user-attachments/assets/7b900aca-6a27-428e-8a9d-1400839c7307" />

```suggestion
                            class="w-12 h-12 mb-2 rounded-lg" />
```